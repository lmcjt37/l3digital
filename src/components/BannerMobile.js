import React from 'react'
import PropTypes from 'prop-types'

import UtilsHelper from 'helpers/utils'

const BannerMobile = props => (
    <section id="banner" className={`major ${UtilsHelper.getBannerClass()}`}>
        <div className="inner">
            <header className="major">
                <h1>We Are L3 Digital</h1>
            </header>
            <div className="content">
                <p>
                    A team of great collaborators, providing solutions to all
                    your development needs.
                </p>
                <ul className="actions">
                    <li>
                        <a
                            href="javascript:;"
                            className="button next"
                            onClick={props.scrollToElement}
                        >
                            Learn more
                        </a>
                    </li>
                </ul>
            </div>
        </div>
    </section>
)

BannerMobile.propTypes = {
    scrollToElement: PropTypes.func,
}

export default BannerMobile
