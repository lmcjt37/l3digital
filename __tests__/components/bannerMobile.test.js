import React from 'react'
import renderer from 'react-test-renderer'

import BannerMobile from '../../src/components/BannerMobile'

jest.mock('../../src/helpers/utils')

describe('BannerMobile tests', () => {
    it('check it renders correctly with snapshot', () => {
        const tree = renderer.create(<BannerMobile />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
