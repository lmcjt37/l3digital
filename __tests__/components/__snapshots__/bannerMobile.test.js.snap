// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerMobile tests check it renders correctly with snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="major"
      >
        <div
          class="inner"
        >
          <header
            class="major"
          >
            <h1>
              We Are L3 Digital
            </h1>
          </header>
          <div
            class="content"
          >
            <p>
              A team of great collaborators, providing solutions to all your development needs.
            </p>
            <ul
              class="actions"
            >
              <li>
                <a
                  class="button next"
                  href="javascript:;"
                >
                  Learn more
                </a>
              </li>
            </ul>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="major"
    >
      <div
        class="inner"
      >
        <header
          class="major"
        >
          <h1>
            We Are L3 Digital
          </h1>
        </header>
        <div
          class="content"
        >
          <p>
            A team of great collaborators, providing solutions to all your development needs.
          </p>
          <ul
            class="actions"
          >
            <li>
              <a
                class="button next"
                href="javascript:;"
              >
                Learn more
              </a>
            </li>
          </ul>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
